ShipEngine/ShipEngineClient.cs

```diff
@@ -71,6 +74,39 @@ public static HttpClient ConfigureHttpClient(Config config, HttpClient client)
             return client;
         }
 
+        /// <summary>
+        /// Sets the HttpClient User agent, the json media type, and the API key to be used
+        /// for all ShipEngine API calls unless overwritten at the method level.
+        /// </summary>
+        /// <param name="client">The HttpClient to be configured</param>
+        /// <param name="apiKey">The API key to be used for all ShipEngine API calls</param>
+        /// <param name="baseUri">The base URI for the ShipEngine API</param>
+        /// <param name="timeout">The timeout for the ShipEngine API Calls</param>
+        /// <returns></returns>
+        public static HttpClient ConfigureHttpClient(HttpClient client, string apiKey, Uri? baseUri, TimeSpan? timeout = null)
+        {
+            client.DefaultRequestHeaders.Accept.Clear();
+
+            var osPlatform = Environment.OSVersion.Platform.ToString();
```

Comment:
The duplication of all of this logic seems wrong. Can the non-static version invoke the static version?

Reply:
good callout, I have modified this in the updated PR here: https://github.com/ShipEngine/shipengine-dotnet/pull/93